core/src/main/java/org/wildfly/extras/sunstone/api/impl/ObjectType.java

```diff
@@ -6,7 +6,8 @@
  */
 public enum ObjectType implements ObjectPropertiesType {
     CLOUD_PROVIDER("cloud.provider", "cloud provider"),
-    NODE("node", "node");
+    NODE("node", "node"),
+    JUNIT5("junit5", "JUnit5");
```

Comment:
I don't think JUnit5 belongs here. I understand why it's here (one configuration for the entire extension), but I think it's going to cause trouble once we try to deploy in multiple regions, under multiple users, etc.